src/consume.ts

```diff
@@ -1027,13 +1058,3 @@ class Timer extends Data {
     return this.copy({ start: Date.now(), offset: 0 });
   }
 }
-
-function track(details: object) {
```

Comment:
Move these up into the main closure so that can then close over the topic being viewed.